### DISCUSSION TITLE
Is there a size limit to notion2anki ?

### DISCUSSION THREAD

**Discussion by Mr-Folder** [2anki/server#1363] [Q&A]
Hello, first, thanks for the wonderful work. 

I have been trying export to my workspace (30mb file), but the server will just load forever.

Eventually, I received an "(node:1) MaxListenersExceededWarning: Possible EventEmitter memory leak detected.  Close listeners added to [Server]. Use emitter.setMaxListeners() to increase limit". I don't know if this is related

Thanks :)

**Comment by aalemayhu** [comment]
Thanks for the report. I will look into.

**Comment by aalemayhu** [comment]
Yes, there are limits in place but closing this since it has been increased to 100mb, for more details see https://docs.2anki.net/troubleshooting/limits/
